test/char.jl

```diff
@@ -1,6 +1,40 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
 #tests for /base/char.jl
+
+@test typemin(Char) == 0
+@test ndims(Char) == 0
+@test getindex('a', 1) == 'a'
+@test_throws BoundsError getindex('a',2)
+# This is current behavior, but it seems incorrect
+@test getindex('a',1,1,1) == 'a'
+@test_throws BoundsError getindex('a',1,1,2)
+# This gets a deprecation warning that seems incorrect
+# getindex('a',1.0,1) -> WARNING: indexing with non Integer Reals is deprecated
+# Capture spurious deprecation warning
+let
```

Comment:
Again, please remove this.  You can leave a note to update when the implementation changes, but for now just test what is currently implemented.  This is not going to change for 0.4.